exercises/verify.cpp

```diff
@@ -2,17 +2,33 @@
 Write a software that verifies if a number is present in a pre-defined array.
 
 Output example:
-Insert number 3
+nsert numberI 3
 The number 3 is [not] present in the array.
 */
 
 #include <iostream>
 using namespace std;
 
-int main()
-{
+int main() {
+  int number;
+  cout << "nsert number: ";
+  cin >> number;
+
   // placeholder
   int N[10] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
```

Comment:
Meglio usare una macro per i valori handcoded , come lo 0 nel return o la dimensione dell'array.
Questo perchè le macro create con #define non una variabile perchè occupa memora, il simbolo di macro viene sostituito con il valore reale in fase di compilazione, per info: https://www.arduino.cc/reference/en/language/structure/further-syntax/define/:
```diff
+ #define ARRAY_DIM 10
....
+ int N[ARRAY_DIM]
- int N[10]
```